storage/mysql/log_storage_test.go

```diff
@@ -977,38 +977,34 @@ func TestGetUnsequencedCounts(t *testing.T) {
 	s := NewLogStorage(DB, nil)
 
 	ctx := context.Background()
-	expectedCount := int64(0)
+	expectedCount := make(map[int64]int64)
 
 	for i := int64(1); i < 10; i++ {
```

Comment:
That doesn't test that the `GetSequencedCounts()` will actually notice ongoing changes in the queue size - this is the point of the outer loop.